app/helpers/stream_entries_helper.rb

```diff
@@ -51,7 +51,9 @@ def rtl?(text)
     rtl_characters = /[\p{Hebrew}|\p{Arabic}|\p{Syriac}|\p{Thaana}|\p{Nko}]+/m.match(text)
 
     if rtl_characters.present?
-      total_size = text.strip.size.to_f
+      # Remove mentions before counting characters to decide RTL ratio
```

Comment:
Probably, but I don't have time to learn the lingo at the moment. Can I promise to add that at a later PR? :)